neo/test/iotest/common_io_test.py

```diff
@@ -192,65 +194,6 @@ def generic_io_object(self, filename=None, return_path=False, clean=False):
                                         return_path=return_path,
                                         clean=clean)
 
-    def create_file_reader(self, filename=None, return_path=False,
```

Comment:
I don't remember why we add this before.
it is not used anymore ?

Reply:
It is not used within Neo and is also not working, e.g. when trying to create a generic reader as the default filename does not exist.